jax/random.py

```diff
@@ -277,7 +277,7 @@ def _random_bits(key, bit_width, shape):
     # TODO(mattjj): just split the key here
     raise TypeError("requesting more random bits than a single call provides.")
 
-  counts = lax.tie_in(key, lax.iota(onp.uint32, max_count))
+  counts = lax.tie_in(key, lax.iota(onp.uint32, max_count.astype(onp.uint32)))
```

Comment:
Why do we need this change in this PR?

Reply:
We don't (relict from the random implementation taken out of this PR), now removed.